### ISSUE TITLE
2019-12-10 tc106 M1SD intermediary output appears in STDOUT in appropriately

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#97]
commit 910d0dc 
tc106 files loaded to sharepoint

Note that blank dataframe appears before the run_computation call completes:

```
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
   SDEID DOSE CMAX FLGACCEPTPREDOSE TMAX FLGACCEPTTMAX TLAST KEL KELTMLO KELTMHI KELNOPT KELRSQ FLGACCEPTKEL THALF THALFF AUCLAST AUCINFP AUCXPCTP CLFP VZFP CONC1 CONC2 CONC3 CONC4 CONC5 CONC6 CONC7 CONC8 CONC9 CONC10 CONCTIME1 CONCTIME2 CONCTIME3
1     NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
2     NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
3     NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
4     NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
5     NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
6     NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
7     NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
8     NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
9     NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
10    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
11    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
12    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
13    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
14    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
15    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
16    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
17    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
18    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
19    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
20    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
21    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
22    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
23    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
24    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
25    NA   NA   NA               NA   NA            NA    NA  NA      NA      NA      NA     NA           NA    NA     NA      NA      NA       NA   NA   NA    NA    NA    NA    NA    NA    NA    NA    NA    NA     NA        NA        NA        NA
   CONCTIME4 CONCTIME5 CONCTIME6 CONCTIME7 CONCTIME8 CONCTIME9 CONCTIME10
1         NA        NA        NA        NA        NA        NA         NA
2         NA        NA        NA        NA        NA        NA         NA
3         NA        NA        NA        NA        NA        NA         NA
4         NA        NA        NA        NA        NA        NA         NA
5         NA        NA        NA        NA        NA        NA         NA
6         NA        NA        NA        NA        NA        NA         NA
7         NA        NA        NA        NA        NA        NA         NA
8         NA        NA        NA        NA        NA        NA         NA
9         NA        NA        NA        NA        NA        NA         NA
10        NA        NA        NA        NA        NA        NA         NA
11        NA        NA        NA        NA        NA        NA         NA
12        NA        NA        NA        NA        NA        NA         NA
13        NA        NA        NA        NA        NA        NA         NA
14        NA        NA        NA        NA        NA        NA         NA
15        NA        NA        NA        NA        NA        NA         NA
16        NA        NA        NA        NA        NA        NA         NA
17        NA        NA        NA        NA        NA        NA         NA
18        NA        NA        NA        NA        NA        NA         NA
19        NA        NA        NA        NA        NA        NA         NA
20        NA        NA        NA        NA        NA        NA         NA
21        NA        NA        NA        NA        NA        NA         NA
22        NA        NA        NA        NA        NA        NA         NA
23        NA        NA        NA        NA        NA        NA         NA
24        NA        NA        NA        NA        NA        NA         NA
25        NA        NA        NA        NA        NA        NA         NA
 [ reached 'max' / getOption("max.print") -- omitted 50 rows ]
unit_conversion : Unit Class 1 (Time) time_col:  TMAX TLAST KELTMLO KELTMHI THALF  parameters are scaled from  HR  to  HR  via scaling factor:  1 
unit_conversion : Unit Class 3 (Dose) dose_col:  DOSE  parameters are scaled from  MG  to  MG  via scaling factor:  1 
unit_conversion : Unit Class 4 (Volume) volume_col:  VZFP  parameters are scaled from  ML  to  L  via scaling factor:  0.001 
unit_conversion : Unit Class 5 (Amount/Volume) conc_col:  CMAX  parameters are scaled from  NG/ML  to  NG/ML  via scaling factor:  1 
unit_conversion : Unit Class 6 (1/Time) kel_col:  KEL  parameters are scaled from  1/HR  to  1/HR  via scaling factor:  1 
unit_conversion : Unit Class 7 (Volume/Time) cl_col:  CLFP  parameters are scaled from  ML/HR  to  L/HR  via scaling factor:  0.001 
unit_conversion : Unit Class 8: (Amount.Time/Volume) auc_col:  AUCLAST AUCINFP  parameters are scaled from  NG.HR/ML  to  NG.HR/ML  via scaling factor:  1 
Warning messages:
1: In validate_timeconc_data(map, data, verbose = verbose) :
  No tau information provided in 'map'.
2: In validate_timeconc_data(map, data, verbose = verbose) :
  No told information provided in 'map'.
3: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGEMESIS' is not present in the dataset
4: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'AMOUNTU' #2# is not present in the dataset provided via 'map'
5: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'PKPTMU', 'CONCU' and 'DOSEU' #8# are not present in the dataset provided via 'map'
6: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'PKPTMU' and 'CONCU' #10# are not present in the dataset provided via 'map'
7: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'CONCU' #24# is not present in the dataset provided via 'map'
8: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'CONCU', 'PKPTMU and 'NORMBSU' #19# are not present in the dataset provided via 'map'
```

**Comment by jhhughes256** [comment]
### Merging Issues

This also occurs for tc008 M1SD and tc010 M1SD.

**Comment by rudraya** [comment]
This issue was addressed in e6af3b4, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
This issue no longer occurs as of e6af3b4.
